src/test/java/io/datareplication/util/SettableClock.java

```diff
@@ -0,0 +1,57 @@
+package io.datareplication.util;
+
+import lombok.Getter;
+
+import java.time.Clock;
+import java.time.Duration;
+import java.time.Instant;
+import java.time.ZoneId;
+import java.time.ZoneOffset;
+
+@Getter
+public class SettableClock extends Clock {
+    private final Instant initialTime;
```

Comment:
We could use a LocalDateTime to enable the usage of the SettableClock with the RFC_1123_DATE_TIME Formatter

`DateTimeFormatter.RFC_1123_DATE_TIME.format(LocalDateTime.now().atOffset(ZoneOffset.UTC));`

which is not possible with Instant